docs/docs/Integrations/Google/integrations-setup-google-cloud-vertex-ai-langflow.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Integrate Google Cloud Vertex AI with Langflow'
+slug: /integrations-setup-google-cloud-vertex-ai-langflow
+sidebar_position: 2
+description: "A comprehensive guide on creating a Google OAuth app, obtaining tokens, and integrating them with Langflow's Google components."
+---
+
+Langflow integrates with the [Google Vertex AI API](https://console.cloud.google.com/marketplace/product/google/aiplatform.googleapis.com) for authenticating the [Vertex AI embeddings model](/components-embedding-models#vertexai-embeddings) and [Vertex AI](/components-models#vertexai) components.
+
+Learn how to create a service account JSON in Google Cloud to authenticate Langflow’s Vertex AI components.
+
+## Create a service account with Vertex AI access
+
+1. Select and enable your Google Cloud project.
+For more information, see [Create a Google Cloud project](https://developers.google.com/workspace/guides/create-project).
+2. Create a service account in your Google Cloud project.
+For more information, see [Create a service account](https://developers.google.com/workspace/guides/create-credentials#service-account).
+3. Assign the **Vertex AI Service Agent** role to your new account.
+This role allows Langflow to access Vertex AI resources.
+For more information, see [Vertex AI access control with IAM](https://cloud.google.com/vertex-ai/docs/general/access-control).
+4. To generate a new JSON key for the service account, navigate to your service account.
+5. Click **Add Key**, and then click **Create new key**.
+6. Under **Key type**, select **JSON**, and then click **Create**.
+A JSON private key file is downloaded.
+
```

Comment:
Suggest adding a sentence to indicate that they have to go to the next set of instructions to finish the setup. Something like "Now that you have a service account and a JSON private key, you need to configure the credentials in Langflow components."

Reply:
Agreed, added